### PR TITLE
glwiki article quality model

### PR DESCRIPTION
**Pull request by halfak**
https://phabricator.wikimedia.org/T201146

### CODE REVIEW

**Comment by adamwight** [comment]
Lint annoyances:
```
./articlequality/feature_lists/glwiki.py:57:9: E131 continuation line unaligned for hanging indent
./build/lib/articlequality/feature_lists/glwiki.py:57:9: E131 continuation line unaligned for hanging indent
```

**Comment by adamwight** [comment]
Models must be uploaded as LFS objects.

File: Makefile [hunk 1]
```diff
@@ -164,6 +164,64 @@ models/euwiki.wp10.gradient_boosting.model: \
 		--labels '"Stub","Start","C","B","GA","FA"' \
 	  --center --scale > $@
 
+########################## Galician Wikipedia ##################################
```
Comment (adamwight) [inline Makefile]: leading whitespace

File: articlequality/feature_lists/glwiki.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    category_links,
+    category_links / max(wikitext.revision.content_chars, 1),
+    cn_templates + 1,
+    cn_templates / max(wikitext.revision.content_chars, 1),
+    log(paragraphs_without_refs_total_length + 1),
+    paragraphs_without_refs_total_length /
+        max(wikitext.revision.content_chars, 1),
+]
+
+wp10 = wikipedia.article + local_wiki
```
Comment (adamwight) [inline articlequality/feature_lists/glwiki.py]: TODO: s/wp10/articlequality/ one day soon

File: Makefile (hunk 1 above)
Comment (halfak) [reply Makefile]: Where?

File: .gitattributes [hunk 3]
```diff
@@ -1 +1,2 @@
 *.model filter=lfs diff=lfs merge=lfs -text
+models/*.model filter=lfs diff=lfs merge=lfs -text
```
Comment (adamwight) [inline .gitattributes]: Thanks for the catch.  Looks like the first line is nonsense, and should be removed.  If anything, it should have been:

    **/*.model ...

per https://git-scm.com/docs/gitignore#_pattern_format

File: Makefile (hunk 1 above)
Comment (adamwight) [reply Makefile]: You're right, nothing's there.

**Pull request merged**